collectivo/server/api/create_example_data.post.ts

```diff
@@ -297,19 +307,19 @@ async function cleanShiftsData() {
 
   const schemas = [
     "shifts_shifts",
-    "shifts_slots",
     "shifts_assignments",
     "shifts_absences",
     "shifts_logs",
   ];
 
   for (const schema of schemas) {
+    console.log(`    deleting 1000 items in schema ${schema} ...`);
     await directus.request(deleteItems(schema, { limit: 1000 }));
   }
 }
 
-const SHIFT_TIMES_OF_DAY = [10, 13, 16, 19];
-const SHIFT_CYCLE_START = DateTime.local(2024, 1, 1);
+const SHIFT_TIMES_OF_DAY = [8, 11, 14, 17];
+const SHIFT_CYCLE_START = DateTime.local(2025, 4, 7);
```

Comment:
Was wäre wenn du hier immer das heutige Datum nimmt?

Reply:
ach, da hat sich eine Änderung von meinem anderen Branch eingeschlichen. Nicht von Master gebrancht..
Aber guter Punkt, werde ich in https://github.com/MILA-Wien/mila-server/pull/38 aufnehmen!

Reply:
hab ich nun hier umgesetzt: https://github.com/MILA-Wien/mila-server/pull/38/commits/af944f72bc22dc5985c32b8f9847b418bee13d15

Reply:
die Times of Day sind die Uhrzeiten, zu denen die Schichten starten. Die sind einfach angelehnt an unsere echten derzeitigen Schichten konstant um 8, 11, 14 und 17 Uhr.